contracts/handlers/ERC20Handler.sol

```diff
@@ -21,7 +21,7 @@ contract ERC20Handler is IDepositExecute, HandlerHelpers, ERC20Safe {
     }
 
     // destId => depositNonce => Deposit Record
-    mapping (uint8 => mapping(uint64 => DepositRecord)) public _depositRecords;
+    mapping (uint8 => mapping(uint64 => DepositRecord)) public _depositRecords; // REVIEW: Not used on chain, so doesn't have to be stored on chain, emitting an event is enough. Will save ~120000 gas on deposit.
```

Comment:
we need this record on chain to query off chain and construct proposals

Reply:
It is available in the events, so off-chain can read it from there. It is not as convenient, but cheaper.